libs/island-ui/core/src/lib/Button/Button.treat.ts

```diff
@@ -3,7 +3,7 @@ import { theme, themeUtils } from '@island.is/island-ui/theme'
 
 const speed = '150ms'
 
-const hexToRgb = (hex) => {
+const hexToRgb = (hex: string) => {
```

Comment:
Move this over to utils?

And perhaps combine all transition speeds for the core UI library somewhere